pages/search/formatters/index.jsx

```diff
@@ -11,12 +11,15 @@ import format from 'date-fns/format';
 
 const DATE_FORMAT = 'D MMM YYYY';
 
+// markdown bolding breaks if there's a space before the closing markers
+const markdownSafe = highlight => highlight && highlight.replace(' **', '** ');
```

Comment:
this doesn't seem legit - it will only change the first one - you want a global regex probably

Reply:
also - why the trailing space?

Reply:
if you don't have the trailing space it converts `**Anne Marie **Cooper` to `**Anne Marie**Cooper` and renders **Anne Marie**Cooper.

But yeah, I should probably replace all occurances, I always forget `str.replace()` doesn't do that by default